buildroot-external/package/eq3_char_loop/eq3_char_loop.c

```diff
@@ -0,0 +1,996 @@
+/*
+* eQ-3 char loopback driver for HomeMatic / HomeMatic IP dual stack implementations

```

Comment:
Can we get this file from its source directly maybe?

It is not that big, so not big of a deal, but I think it would make it easier to update when there are upstream changes.

Reply:
I will see if I can rework the buildroot package to fetch these sources upstream and then just patch what is missing (e.g. upstream is not 64bit compatible). But yes, in principle they can be fetched upstream.